nvr/nvr.py

```diff
@@ -330,20 +330,33 @@ def main(argv=sys.argv, env=os.environ):
         print_sockaddrs()
         return
 
-    address = options.servername or env.get('NVIM_LISTEN_ADDRESS') or '/tmp/nvimsocket'
+    if sys.platform == 'win32':
+        address = options.servername or env.get('NVIM_LISTEN_ADDRESS') or None
+        if address == None:
+            print("Windows workaround: could not find an address! Set environment variable 'NVIM_LISTEN_ADDRESS', or pass a server address when calling nvr: `nvr --servername \\some\\server\\address`.")
```

Comment:
No exclamation points in messages, and I think the message will look cleaner without the "Windows workaround:" prefix.

Also, it's not a convention for this repo, but I think these lines would look better if they were shortened.  Whether or not you should actually do it is up to @mhinz, though.

Example:

```python
print("Windows workaround: could not find an address! Set "
      "environment variable 'NVIM_LISTEN_ADDRESS', or pass "
      "a server address when calling nvr: "
      "`nvr --servername \\some\\server\\address`.")
```